q2_diversity_lib/plugin_setup.py

```diff
@@ -76,36 +78,168 @@
+    output_descriptions={'vector': "Vector containing per-sample values "
+                                   "for Pielou's Evenness."},
+    name="Pielou's Evenness",
+    description="Compute Pielou's Evenness for each sample in a "
+                "feature table",
     citations=[citations['pielou1966measurement']]
 )
 
+# TODO: Augment citations as needed
 plugin.methods.register_function(
     function=q2_diversity_lib.shannon_entropy,
     inputs={'table': FeatureTable[Frequency | RelativeFrequency]},
     parameters={'drop_undefined_samples': Bool},
     outputs=[('vector',
              SampleData[AlphaDiversity])],
-    input_descriptions={'table': 'The feature table containing the samples '
-                        'for which Shannon\'s Entropy should be computed.'},
-    parameter_descriptions={'drop_undefined_samples': 'Samples with no '
-                            'observed features produce undefined (NaN) values.'
-                            ' If true, these samples are dropped from the '
-                            'output vector.'},
-    output_descriptions={'vector': 'Vector containing per-sample values '
-                                   'for Shannon\'s Entropy.'},
-    name='Shannon\'s Entropy',
-    description='Compute Shannon\'s Entropy for each sample in a '
-                'feature table',
+    input_descriptions={'table': "The feature table containing the samples "
+                        "for which Shannon's Entropy should be computed."},
+    parameter_descriptions={'drop_undefined_samples': "Samples with no "
+                            "observed features produce undefined (NaN) values."
+                            " If true, these samples are dropped from the "
+                            "output vector."},
+    output_descriptions={'vector': "Vector containing per-sample values "
+                                   "for Shannon's Entropy."},
+    name="Shannon's Entropy",
+    description="Compute Shannon's Entropy for each sample in a "
+                "feature table",
     citations=[citations['shannon1948communication']]
 )
+
+# ------------------------ beta-diversity -----------------------
+# TODO: Augment citations as needed
+plugin.methods.register_function(
+    function=q2_diversity_lib.bray_curtis,
+    inputs={'table': FeatureTable[Frequency]},
+    parameters={'n_jobs': Int % Range(1, None) | Str % Choices(['auto'])},
+    outputs=[('distance_matrix', DistanceMatrix)],
+    input_descriptions={
+        'table': "The feature table containing the samples for which "
+                 "Bray-Curtis dissimilarity should be computed."},
+    parameter_descriptions={
+        'n_jobs': "The number of concurrent CPU jobs to use in performing this"
+                  " calculation.  More jobs = faster performance. May not "
+                  "exceed the number of available physical cores. If n_jobs = "
+                  "'auto', all available CPUs are used."},
```

Comment:
Looking good! I'll propose some tweaks here:

```suggestion
        'n_jobs': "The number of concurrent jobs to use in performing this"
                  " calculation. May not "
                  "exceed the number of available physical cores. If n_jobs = "
                  "'auto', one job will be launched for each identified CPU core on the host."},
```

The changes proposed above aren't reformatted because I want the diff to highlight the nature of the changes proposed (as opposed to shuffling due to line-shuffling).

A few notes:

> More jobs = faster performance.

Unfortunately this is generally not the case, and often is the exact opposite at high levels of oversubscription. Complicating matters, oversubscription can occur easy through unrelated processes.

Reply:
Good feedback. I also updated the `threads` description here https://github.com/qiime2/q2-diversity-lib/pull/6/commits/687401d120b697f3fcdf7f5f3f11317b4d6291b3. Mind resolving this conversation if both sections look good?